src/__tests__/defineLambdaTestSuite.ts

```diff
@@ -0,0 +1,53 @@
+import { ApolloServer, ApolloServerOptions, BaseContext } from '@apollo/server';
+import {
+  CreateServerForIntegrationTestsOptions,
+  defineIntegrationTestSuite,
+} from '@apollo/server-integration-testsuite';
+import type { Handler } from 'aws-lambda';
+import { createServer, IncomingMessage, ServerResponse } from 'http';
+import { startServerAndCreateLambdaHandler } from '..';
+import { urlForHttpServer } from './mockServer';
+
+export function defineLambdaTestSuite<Event, Response>(
+  mockServerFactory: (
+    handler: Handler<Event, Response>,
+    shouldBase64Encode: boolean,
+  ) => (req: IncomingMessage, res: ServerResponse) => void,
+) {
+  for (const shouldBase64Encode of [true, false]) {
+    describe(`With base64 encoding ${
+      shouldBase64Encode ? 'enabled' : 'disabled'
+    }`, () => {
```

Comment:
Silly nit but jest provides a construct for this [`describe.each`](https://jestjs.io/docs/api#describeeachtablename-fn-timeout).

I can't say I see an argument for or against so take it or leave it as you please.